### ISSUE TITLE
[Bug] FPS Unlockers don't work in fullscreen

### ISSUE THREAD

**Issue by Meladoom2** [Ryujinx/Ryujinx#4697]
### Description of the issue

I installed 60fps mods for Fire Emblem: Three Houses and Fire Emblem Engage, and they work, except only when playing windowed. But when I open fullscreen (F11, Alt+Enter), Ryujinx locks games at 30fps. 

### Reproduction steps

1 - Install 60fps mod for Fire Emblem: Three Houses and/or Fire Emblem Engage (probably happens in other games)
2 - Start the game in fullscreen mode

### Log file

[Ryujinx_1.1.723_2023-04-18_19-57-32.log](https://github.com/Ryujinx/Ryujinx/files/11264550/Ryujinx_1.1.723_2023-04-18_19-57-32.log)
[Ryujinx_1.1.723_2023-04-18_20-00-27.log](https://github.com/Ryujinx/Ryujinx/files/11264551/Ryujinx_1.1.723_2023-04-18_20-00-27.log)
[Ryujinx_1.1.723_2023-04-18_20-12-15.log](https://github.com/Ryujinx/Ryujinx/files/11264552/Ryujinx_1.1.723_2023-04-18_20-12-15.log)

### OS

Windows 10

### Ryujinx version

1.1.723

### Game version

FE Engage 2.0.0, FE3H 1.2.0

### CPU

Ryzen 5 3600

### GPU

NVIDIA GTX 650 Ti 1GB

### RAM

16GB

### List of applied mods

Any 60FPS mod, such as this
https://gamebanana.com/mods/432758

### Additional context?

Tested with MSI Afterburner + RTSS
It most likely happens because Ryujinx doesn't have Windowed Fullscreen mode.

Disabling/Enabling VSync both in Ryujinx (F1) and Nvidia Control Panel doesn't help

**Comment by MutantAura** [comment]
This is likely something in your driver or monitor settings. Fullscreening will not affect how Ryujinx presents. 
e.g BoTW 60fps mod working just fine in fullscreen:
<img width="1920" alt="image" src="https://user-images.githubusercontent.com/44103205/232856870-9b1b5f3b-6556-40ac-b6e1-2cd36a7d6eae.png">

**Comment by Meladoom2** [comment]
Huh, these mods work flawlessly on Yuzu, so I assumed this is something on Ryujinx's side.
I think I tried all of the generic solutions, and none of them worked (such as disabling VSync and frame limiters in NVCP, and some other things)

**Comment by MikePrime13** [comment]
If I'm reading your hardware correctly, your GPU hardware may not be strong enough to render FE Engage at 60 FPS consistently.

I've been playing FE Engage on Ryujinx extensively since it came out, and I have several machines I play on. I wasn't able to hit consistent 60 FPS on an RTX 2070 mobile and I did get some FPS drops (below 30) on a GTX 1080. However, I was able to get a more consistent 40+ FPS on RTX 2070 Super on a desktop, and on a new 4070, the 60FPS is more or less stable unless I hit some parts of Somniel that is simply the game not being optimized in my view.

**Comment by MutantAura** [comment]
Doesn't seem like anyone else is able to reproduce. Closing, sorry.

**Comment by Meladoom2** [comment]
Update: This happens only when using OpenGL. And only when using ANY of the Post Processing Effects (FXAA or any of the SMAA). If you select "None" everything's good. (Temporary fix?)
Tested on version 1.1.751 in FE3H and FE Engage

Just in case, I'm using OpenGL because Vulkan is basically unusable for some reason (Couldn't find any info on google about 650Ti1GB not supporting it)

Sorry for reviving this thread, I did found a fix and wanted to leave it here
